packages/admin/admin/src/section/SectionHeadline.tsx

```diff
@@ -0,0 +1,138 @@
+import { Info } from "@comet/admin-icons";
+import { type ComponentsOverrides, Divider, Typography } from "@mui/material";
+import { css, type Theme, useThemeProps } from "@mui/material/styles";
+import { type ReactNode } from "react";
+
+import { Tooltip } from "../common/Tooltip";
+import { createComponentSlot } from "../helpers/createComponentSlot";
+import { type ThemedComponentBaseProps } from "../helpers/ThemedComponentBaseProps";
+
+export type SectionHeadlineClassKey = "root" | "headlineWrapper" | "titleContainer" | "divider" | "supportText" | "infoTooltipText" | "infoIcon";
```

Comment:
Would `header` be a more fitting name for `headlineWrapper`? 
As it doesn't only wrap the headline but also the icon and the support-text.